linkedevents/management/commands/check_migrations.py

```diff
@@ -0,0 +1,35 @@
+from django.db.migrations.loader import MigrationLoader
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """
+    Note, to use you probably need to set EXTRA_INSTALLED_APPS=linkedevents
```

Comment:
So do I need to or not? "Probably" isn't a good enough answer; please elaborate. For example, why? When?